examples/next/faustwp-getting-started/package.json

```diff
@@ -23,7 +23,7 @@
     "sass": "^1.54.9"
   },
   "engines": {
-    "node": ">=16",
+    "node": ">=18",
```

Comment:
Is there any downside to bumping to 20? It's the most recent LTS version.

Reply:
I wouldn't just because 18 is still a relevant and non deprecated version.